browser-interface/packages/shared/portableExperiences/sagas.ts

```diff
@@ -38,7 +48,39 @@ export function* portableExperienceSaga(): any {
     handlePortableExperienceChanges
   )
   yield takeEvery(RELOAD_SCENE_PX, reloadPortableExperienceChanges)
+
+  // Debounce it -- this is likely due to some internal logical bug
+  // TODO: Figure out if we still need this
```

Comment:
Many actions can trigger a rebuild of the portable experience "desired list", i.e. adding 1 by one a wearable with a PX because of the async loading of wearables. The debounce exists only to prevent sending many similar messages to the renderer and to prevent the associated serialization/processing cost